### ISSUE TITLE
🥛 Make app navigation semi-transparent

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/nextcloud-vue#3047]
As per the mockups

![Bildschirmfoto vom 2022-08-16 12-58-35](https://user-images.githubusercontent.com/1374172/184863757-c87f5d04-c559-4c29-b132-a21b720c8f43.png)

the app navigation component should become semi-transparent and let the background image show through with a blur.

## Specification

- 80% opaque white with 10px blur, like in Dashboard. Code: `background-color: var(--color-background-translucent); -webkit-backdrop-filter: var(--background-blur); backdrop-filter: var(--background-blur);`
- [AppNavigationNew](https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/AppNavigationNew/AppNavigationNew.vue) gets our Nextcloud fade, looks nice and clicky
- WCAG AA: Contrast of secondary text in the navigation works after darkening it to `5c5c5c`
- WCAG AAA: Just like for Dashboard, the high contrast theme should make the navigation background fully opaque and the secondary text color also color-main-text.
